core/templates/components/entity-creation-services/story-creation.service.ts

```diff
@@ -16,93 +16,72 @@
  * @fileoverview Modal and functionality for the create story button.
  */
 
-require(
-  'pages/topic-editor-page/modal-templates/' +
-  'create-new-story-modal.controller.ts');
+import { downgradeInjectable } from '@angular/upgrade/static';
+import { Injectable } from '@angular/core';
+import { AlertsService } from 'services/alerts.service';
+import { ImageLocalStorageService } from 'services/image-local-storage.service';
+import { LoaderService } from 'services/loader.service';
+import { UrlInterpolationService } from 'domain/utilities/url-interpolation.service';
+import { NgbModal } from '@ng-bootstrap/ng-bootstrap';
```

Comment:
Please put any imports that begin with `@` at the top of the list. It may also be a good idea to put a new line after the last `@` import for easier readability!

Reply:
Done.